### PR TITLE
[Framework]:Initial folder structure.

### PR DESCRIPTION
**Pull request by LingerDavid**
This is the initial folder structure referring to the original version of transformer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,4 +118,26 @@ graph LR
...
\ No newline at end of file
+```
+
+
+## The Folder Structure
+The folder of the project is desigened by slight structure for expansibility and convenience:
+```shell
+.
+├── data # mounted data and scripts
+│   ├── data_lodar.py
```
Comment (eleven-yun) [inline README.md]: data_loader?

File: README.md [hunk 2]
```diff
@@ -118,4 +118,26 @@ graph LR
...
+│   └── nas
```
Comment (eleven-yun) [inline README.md]: Do we need or do we plan to use network attached storage for the PoC stage?

File: README.md [hunk 3]
```diff
@@ -118,4 +118,26 @@ graph LR
...
+├── eval.py # evaluation scipts
+├── models # transformer modules
+│   ├── decoder
+│   └── encoder
+├── README.md
+├── scripts #  scripts for convenience
```
Comment (eleven-yun) [inline README.md]: 1. I think we can put train.py, test.py and eval.py into scripts/ folder.
2. The comment does not seem necessary, consider remove.

File: README.md [hunk 4]
```diff
@@ -118,4 +118,26 @@ graph LR
...
+
+## The Folder Structure
+The folder of the project is desigened by slight structure for expansibility and convenience:
+```shell
+.
+├── data # mounted data and scripts
+│   ├── data_lodar.py
+│   └── nas
+├── eval.py # evaluation scipts
+├── models # transformer modules
```
Comment (eleven-yun) [inline README.md]: 1. models -> nn_model or nn_modules
2. The comment does not seem necessary, consider remove.

File: README.md [hunk 5]
```diff
@@ -118,4 +118,26 @@ graph LR
     iadd-->enc_sa;
 
     softmax--"Probablity"-->pred;
-```
\ No newline at end of file
+```
+
+
+## The Folder Structure
+The folder of the project is desigened by slight structure for expansibility and convenience:
```
Comment (eleven-yun) [inline README.md]: Consider rewrite this sentence, which could be as simple as:
"The files of this project are organized as follows."

File: README.md [hunk 6]
```diff
@@ -118,4 +118,26 @@ graph LR
...
+```shell
+.
+├── data # mounted data and scripts
+│   ├── data_lodar.py
+│   └── nas
+├── eval.py # evaluation scipts
```
Comment (eleven-yun) [inline README.md]: The comment does not seem necessary, consider remove.

File: README.md [hunk 7]
```diff
@@ -118,4 +118,26 @@ graph LR
...
+├── models # transformer modules
+│   ├── decoder
+│   └── encoder
+├── README.md
+├── scripts #  scripts for convenience
+├── test.py # test interface
```
Comment (eleven-yun) [inline README.md]: The comment does not seem necessary, consider remove.

File: README.md [hunk 8]
```diff
@@ -118,4 +118,26 @@ graph LR
...
+├── train.py # train interface
```
Comment (eleven-yun) [inline README.md]: The comment does not seem necessary, consider remove.

File: README.md [hunk 9]
```diff
@@ -118,4 +118,26 @@ graph LR
...
+└── utils # misc functions
```
Comment (eleven-yun) [inline README.md]: The comment does not seem necessary, consider remove.

File: README.md [hunk 10]
```diff
@@ -118,4 +118,26 @@ graph LR
...
-```
\ No newline at end of file
+```
+
+
+## The Folder Structure
+The folder of the project is desigened by slight structure for expansibility and convenience:
+```shell
+.
+├── data # mounted data and scripts
```
Comment (eleven-yun) [inline README.md]: The comment does not seem necessary, consider remove.

**Review by eleven-yun — CHANGES_REQUESTED** [review]
Looks good, except for some minor issues.

File: README.md (hunk 1 above)
Comment (LingerDavid) [reply README.md]: WTF?Forgive me.

File: README.md (hunk 2 above)
Comment (LingerDavid) [reply README.md]: Maybe.Remove it is ok.

File: README.md (hunk 3 above)
Comment (LingerDavid) [reply README.md]: Make sense

File: README.md (hunk 4 above)
Comment (LingerDavid) [reply README.md]: Done.

File: README.md (hunk 5 above)
Comment (LingerDavid) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (LingerDavid) [reply README.md]: Done

File: README.md (hunk 7 above)
Comment (LingerDavid) [reply README.md]: Done

File: README.md (hunk 8 above)
Comment (LingerDavid) [reply README.md]: Done

File: README.md (hunk 9 above)
Comment (LingerDavid) [reply README.md]: Done

File: README.md (hunk 10 above)
Comment (LingerDavid) [reply README.md]: Done

**Review by eleven-yun — APPROVED** [review]
Looks good to me!

**Pull request merged**